client/ElementRegistry.js

```diff
@@ -18,12 +18,16 @@ export default class ElementRegistry {
   }
 
   async initialize() {
-    await Promise.map(elementList, location => this.load(location));
+    await Promise.map(elementList, (location, position) => {
+      return this.load(location, position);
+    });
     const extensions = await this.loadExtensionList();
-    await Promise.map(extensions, location => this.load(location, true));
+    await Promise.map(extensions, (location, position) => {
+      return this.load(location, elementList.length + position, true);
+    });
   }
 
-  async load(location, isExtension) {
+  async load(location, position, isExtension) {
```

Comment:
Move isExtension and position to options object. Set position to last as a default.